### PR TITLE
Feature/checkout page

### PR DESCRIPTION
**Pull request by tomshudev**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -20,6 +21,10 @@ const routes: Routes = [
       },
     ],
   },
+  {
+    path: 'checkout',
+    component: CheckoutComponent,
```
Comment (Sagie501) [inline src/app/app-routing.module.ts:26]: Add AuthGurad

File: src/app/core/components/sidebar/sidebar.component.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <div class="sidebar">
   <a routerLink="/home">Home</a>
   <a routerLink="/user/leasing-history">User Leasing History</a>
+  <a routerLink="/checkout">Checkout</a>
```
Comment (Sagie501) [inline src/app/core/components/sidebar/sidebar.component.html:4]: Shouldn't be in the sidebar, no?

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (tomshudev) [reply src/app/app-routing.module.ts:48]: Done

File: src/app/core/components/sidebar/sidebar.component.html (hunk 2 above)
Comment (tomshudev) [reply src/app/core/components/sidebar/sidebar.component.html]: You right, I have deleted it

**Pull request merged**